src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```diff
@@ -48,10 +55,16 @@
  */
 class AccountController extends AbstractRestController implements ClassResourceInterface, SecuredControllerInterface
 {
+    /**
+     * @deprecated Use the AccountInterface::RESOURCE_KEY constant instead
+     */
     protected static $entityKey = 'accounts';
```

Comment:
Would set the value to `AccountInterface::RESOURCE_KEY`

Reply:
In my opinion, this encourages even more to stick to the `entityKey`, for the other entities we didn't change that as far as I know

Reply:
I did it until now, don't know about the others 🙈

Reply:
think it should not make a big difference. important thing for me is the `@deprecated` 🙂